.travis.yml

```diff
@@ -17,16 +14,11 @@ install:
 - travis_retry pip install sphinx_rtd_theme
 script:
 -   ./ci/run_tests.sh
-- cd docs
-- make html
```

Comment:
i prefer keeping the dOc build script verbOse in travis CI

Reply:
me no preference either but now that it was there i used. rightnow it is serving two other purposes in one place, 1) installing pip requirements of docs 2) adding .nojekyll to build html. 

yaa i do think we should stage the travis nicely, right now i dont know still have to figure that. 
staging should also solve issue not to build docs and updating gh-pages for every python version.